DataLink.tex

```diff
@@ -944,7 +944,7 @@ \subsection{Example: Service Descriptor for VOSpace-2.0}
    </RESOURCE>
 \end{verbatim}
 Since the capability being described is RESTful and inputs are
```

Comment:
For the vospace service descriptor example, I would just remove "and inputs are fixed" because it doesn't add anything.